packages/calcite-components/src/components/tile/tile.scss

```diff
@@ -1,39 +1,59 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-tile-background-color: Specifies the background color of the component.
+ * @prop --calcite-tile-border-color: Specifies the border color of the component.
+ * @prop --calcite-tile-border-color-active: Specifies the border color of the component when in an active state.
```

Comment:
I suppose this will be resolved with upcoming direction re: token construction, but this could potentially be set like `calcite-tile:hover {  --calcite-tile-border-color: pink }`. I would think component-specific props for pseudo classes might be limited to internal components the external developer doesn't have ability to set? Applies to other props.

Reply:
Ok, cool, I've addressed this in the new #8691 pr.